clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/ClinvarContants.scala

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.monster.etl.clinvar
+
+import upack._
+
+/** Container for constants that we use across the ClinVar pipeline. */
+object ClinvarContants {
```

Comment:
The file and object are named `ClinvarContants`, I'm guessing you meant `ClinvarConstants` with an "s" in there!